pythainlp/util/keyboard.py

```diff
@@ -103,6 +103,20 @@
 EN_TH_TRANSLATE_TABLE = str.maketrans(EN_TH_KEYB_PAIRS)
 TH_EN_TRANSLATE_TABLE = str.maketrans(TH_EN_KEYB_PAIRS)
 
+TIS_820_2538 = [
+  "-", "ๅ", "/", "_", "ภ", "ถ", "ุ", "ึ", "ค", "ต", "จ", "ข", "ช",
```

Comment:
`"-", "ๅ", "/", "_"` -> `"_", "ๅ", "/", "-"`

Reply:
It's fixed.